src/editor/components/list/list-item-status.tsx

```diff
@@ -0,0 +1,27 @@
+import { CheckCircleIcon, XCircleIcon } from '@heroicons/react/solid'
+
+interface ListItemStatusProps {
+  status: string
+}
+
+export const ListItemStatus = ( { status }: ListItemStatusProps ): JSX.Element => {
+  const getStatusIcon = ( ): JSX.Element => {
+    switch ( status ) {
+      case 'Active':
+        return <CheckCircleIcon className="text-emerald-400 h-4 w-4" />
+      case 'Disabled':
+        return <XCircleIcon className="text-red-400 h-4 w-4" />
+    }
```

Comment:
should we do status.lower() and check agauinst 'acctive' and 'disabled' to remove case sensitivity?

Reply:
For now I think it's fine to leave it like this. Later we will use an ENUM for this anyway.

Reply:
ok, we'll do this in a refactor / feature